controllers/BooksIndexController.js

```diff
@@ -0,0 +1,38 @@
+angular
+  .module('booksApp')
+  .controller('BooksIndexController', BooksIndexController);
+
+var db = "https://super-crud.herokuapp.com/books"
+
+BooksIndexController.$inject = ['$http'];
+function BooksIndexController(   $http ){
+
+  var vm = this;
+
+  vm.books = [];
```

Comment:
This is fine! You probably need some local storage of the things you've fetched from the db.